### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by FlorianLudwig** [GreyRook/scrum2trello#2]
Probably the same issue experiences by others over at https://github.com/Q42/TrelloScrum/issues/110

**Comment by FlorianLudwig** [comment]
fixed upstream at https://github.com/Q42/TrelloScrum/pull/111

**Comment by SeanColombo** [comment]
Correct, same issue.  Hadn't realized there was a fix until seeing this thread so I went off and chased it myself. Doh!
Here's a commit that includes this fix (and some cleanup I did while I was poking around in there): https://github.com/Q42/TrelloScrum/commit/9deb33d57444542c39f4ef6c8023b57fb8cfd7a4

**Comment by itsbjoern** [comment]
@SeanColombo Hey, great to see you working on TrelloScrum again. Do you think it would be possible to merge our features into the project? We made consumed points optional and were able to remove the delay of showing the picker when opening a card. It would fit into the current project instead of being located in its own fork. If you are interested I will prepare a pull request based on your current version.
